beacon_node/beacon_chain/src/snapshot_cache.rs

```diff
@@ -92,6 +114,7 @@ pub enum StateAdvance<T: EthSpec> {
 }
 
 /// The item stored in the `SnapshotCache`.
+#[derive(Clone)]
```

Comment:
We could even underive Clone now that we're trying to avoid cloning whole CacheItems